Makefile

```diff
@@ -344,6 +344,11 @@ set-manifest-pull-policy:
 ## Deploying
 ## --------------------------------------
 
+#Deploy the BaremetalHost CRDs and CRs (for testing purposes only)
```

Comment:
Do we keep this test item? If yes, Please add a space atfer "#"

Reply:
Thanks @namnx228 , fixed.